medicinskij-academicheskij-jurnal.csl

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Медицинский академический журнал (Russian)</title>
+    <id>http://www.zotero.org/styles/medicinskij-academicheskij-jurnal</id>
+    <link href="http://www.zotero.org/styles/medicinskij-academicheskij-jurnal" rel="self"/>
+    <link href="http://protect.gost.ru/document.aspx?control=7&amp;id=173511" rel="documentation"/>
+    <author>
+      <name>Valery Mukhin</name>
+      <email>Valery.Mukhin@gmail.com</email>
+      <uri>http://sites.google.com/site/yurkin/</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <summary>This style is developed on the base of the &quot;gost-r-7-0-5-2008-numeric&quot  by Yurkin Maxim;, and the main difference is that in this style the delimiters are dots with dashs. And all authors are included in bibliograohies. !!! IMPORTANT !!! Citations are numeric, order of bibliography is in order of citations, default locale is 'en'.    РЎС‚РёР»СЊ РѕСЃРЅРѕРІР°РЅ РЅР° СЃС‚РёР»Рµ &quot;gost-r-7-0-5-2008-numeric&quot;. РџРѕСЏСЃРЅРµРЅРёСЏ Рє РёСЃС…РѕРґРЅРѕРјСѓ СЃС‚РёР»СЋ СЃР»РµРґСѓСЋС‰РёРµ: РћСЃРЅРѕРІРЅР°СЏ РїСЂРѕР±Р»РµРјР° СЃРµР№С‡Р°СЃ - СЌС‚Рѕ Р»РѕРєР°Р»РёР·Р°С†РёСЏ С‚РµСЂРјРёРЅРѕРІ (РЎ., Рў., РїРѕРґ СЂРµРґ.,Рё РґСЂ. ... РёР»Рё СЃРѕРѕС‚РІРµС‚СЃС‚РІРµРЅРЅРѕ P., Vol., ed., et al.). Р’ РїСЂРёРЅС†РёРїРµ СЌС‚Рѕ СЂР°Р±РѕС‚Р°РµС‚ Рё РєРѕРЅС‚СЂРѕР»Р»РёСЂСѓРµС‚СЃСЏ РїРѕР»РµРј  default-locale (РІС‹С€Рµ, РІРѕ РІС‚РѕСЂРѕР№ СЃС‚СЂРѕС‡РєРµ). Р”Р»СЏ СЂСѓСЃСЃРєРѕРіРѕ РЅР°РґРѕ Р·Р°РјРµРЅРёС‚СЊ РЅР° &quot;ru-RU&quot;. Р­С‚Рѕ РѕРїСЂРµРґРµР»СЏРµС‚ РѕРґРЅРё РЅР°СЃС‚СЂРѕР№РєРё РґР»СЏ РІСЃРµР№ Р±РёР±Р»РёРѕРіСЂР°С„РёРё. РћРґРЅР°РєРѕ, РёРґРµР°Р»СЊРЅРѕРµ СЂРµС€РµРЅРёРµ - СЌС‚Рѕ РІС‹Р±РёСЂР°С‚СЊ СЏР·С‹Рє Р»РѕРєР°Р»РёР·Р°С†РёРё С‚РµСЂРјРёРЅРѕРІ РІ Р·Р°РІРёСЃРёРјРѕСЃС‚Рё РѕС‚ СЏР·С‹РєР° Р·Р°РїРёСЃРё (РєР°Рє Рё РїСЂРµРґРїРёСЃС‹РІР°РµС‚ Р“РћРЎРў). Р­С‚Рѕ РЅРµРІРѕР·РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… Zotero 4.0 Рё CSL 1.0, РЅРѕ СѓР¶Рµ СЃРµР№С‡Р°СЃ С‡С‚Рѕ-С‚Рѕ РјРѕР¶РЅРѕ СЃРґРµР»Р°С‚СЊ РІ СЂР°РјРєР°С… MultiLingual Zotero - СЃРј. РЅР°РїСЂРёРјРµСЂ http://forums.zotero.org/discussion/18482/multilingual-zotero-fields-and-csl-variables.    Р РїРѕСЃР»РµРґРЅРµРµ, РїР°С‚РµРЅС‚С‹ РїРѕ СѓРјРѕР»С‡Р°РЅРёСЋ РїСЂРµРґСЃС‚Р°РІР»РµРЅС‹ РєР°Рє USA, РїРѕСЃРєРѕР»СЊРєСѓ РІ CSL 1.0 РЅРµС‚ РїРµСЂРµРјРµРЅРЅРѕР№, СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‰РµР№ РїРѕР»СЋ country РІ Р±Р°Р·Рµ Zotero 4.0. РќР°РґРµСЋСЃСЊ, СЌС‚Рѕ Р±СѓРґРµС‚ РёСЃРїСЂР°РІР»РµРЅРѕ РІ Р±СѓРґСѓС‰РёС… РІРµСЂСЃРёСЏС….</summary>
```

Comment:
@ValeryMukhin, okay. I would like to change this summary text so that it is correctly displayed. Is the following correct? If not, could you copy and paste the Russian text here?

> Стиль основан на стиле "gost-r-7-0-5-2008-numeric". Пояснения к исходному стилю следующие: Основная проблема сейчас - это локализация терминов (С., Т., под ред.,и др. ... или соответственно P., Vol., ed., et al.). В принципе это работает и контроллируется полем default-locale (выше, во второй строчке). Для русского надо заменить на "ru-RU". Это определяет одни настройки для всей библиографии. Однако, идеальное решение - это выбирать язык локализации терминов в зависимости от языка записи (как и предписывает ГОСТ). Это невозможно сделать в рамках Zotero 4.0 и CSL 1.0, но уже сейчас что-то можно сделать в рамках MultiLingual Zotero - см. например http://forums.zotero.org/discussion/18482/multilingual-zotero-fields-and-csl-variables. � последнее, патенты по умолчанию представлены как USA, поскольку в CSL 1.0 нет переменной, соответствующей полю country в базе Zotero 4.0. Надеюсь, это будет исправлено в будущих версиях.

(produced from the ""Р’ РїСЂРёРЅС†РёРїРµ" text with https://2cyr.com/decode/?lang=en)